docker-compose.yml

```diff
@@ -25,9 +26,12 @@ services:
       - "9966:9966"
 
   frontend:
-    image: frontend:latest
+    build:
+      context: ./frontend
     ports:
       - "80:80"
+    volumes:
```

Comment:
This is fine, but you could also just copy nginx.conf file to /etc/nginx/conf.d/default.conf. Approving.